src/connect.js

```diff
@@ -20,7 +20,13 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  const givenParams = params;
+
+  return (extraParams) => {
```

Comment:
одному аргументу скобки не нужны
```suggestion
  return extraParams => {
```